File: week11/week11-2.py
#week11-2py
BOUND = 300000
table = [True] * BOUND
ans = 0
for i in range(2,BOUND):
  if table[i]==True:
    ans += 1
    for k in range(i*i,BOUND,i):
      table[k]=False
print('質數有:', ans, '個')

File: week11/week11-2.cpp
#include <stdio.h>
int table[20000]={};
int main()
{
    int BOUND = 20000, ans = 0;
    for(int i=2;i<BOUND;i++)
    {
        if(table[i]==0)
        {
            ans++;
            for(int k=i*i;k<BOUND;k+=i)
            {
                table[k] =-1;
            }
        }
    }
    printf("½è¼Æ¦³:%d­Ó\n", ans);
}
